OneCommunityGlobal/HighestGoodNetworkApp
# Ramya Unit Test :UsersProfile/ ToolTips/TabToolTips

# Description
Unit test for src/components/UserProfile/ToolTips/TabToolTips

## Main changes explained:
-  New files
      - **_TabtoolTips.test.jsx_** : unit test cases 
      -  _**MockNavLink**_ :  Mock component for the test cases   
- Existing file 
      - **_TabtoolTips_**  : Added  data-attribute to the objects 


## How to test:
1. check into current branch
2. do npm test TabToolTips.test.jsx
3. check if all the test cases pass

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/89441435/ad7aabd9-790b-4a98-9d44-bf9898b49685)


## Note : 
Kindly take a look at the test cases and  suggest if more validations can be added
